core/iwasm/libraries/lib-socket/src/wasi/wasi_socket_ext.c

```diff
@@ -136,7 +136,9 @@ wasi_addr_to_sockaddr(const __wasi_addr_t *wasi_addr,
 int
 accept(int sockfd, struct sockaddr *addr, socklen_t *addrlen)
 {
-    __wasi_addr_t wasi_addr = { 0 };
+    __wasi_addr_t wasi_addr;
+    memset(&wasi_addr, 0, sizeof(wasi_addr));
```

Comment:
Please put memset after line 144 (had better make it a line with an empty line above and below), here it is C source code, we usually align with the C coding style.